gulp-tasks/test.js

```diff
@@ -38,25 +37,25 @@ gulp.task('download-browsers', function() {
   });
 });
 
-gulp.task('mocha', ['build', 'download-browsers'], () => {
-  const mochaOptions = {};
+gulp.task('mocha', ['build'], () => {
+  const mochaOptions = {
+    env: {
+      projectOrStar: global.projectOrStar,
+    },
+  };
+
   if (global.cliOptions.grep) {
     mochaOptions.grep = global.cliOptions.grep;
   }
-  const testGlob = `packages/${global.projectOrStar}/test/*.js`;
-  const testFiles = glob.sync(testGlob);
-  if (testFiles.length === 0) {
-    // Mocha fails when no tests are found so return early.
-    return;
-  }
-  return gulp.src(testGlob, {read: false})
+
+  return gulp.src('test/bootstrap.js', {read: false})
```

Comment:
@jeffposnick Before this PR - yes it would have been expected because it's normally an issue with selenium web driver. With these tests - yes and no. Its going to put the tests under a different process - so may reveal some lucky situations that are no longer true - like the bg-sync-queue tests